### ISSUE TITLE
Default "Original" size of the photo is not available anymore

### ISSUE THREAD

**Issue by mirfilip** [lmammino/flickr-set-get#41]
`Original` is used as the default label of the photo to download. Unfortunately, it looks like such a label is no longer a thing. I tried couple of albums and it usually returns:
```
"Square"
"Large Square"
"Thumbnail"
"Small"
"Small 320"
"Medium"
"Medium 640"
"Medium 800"
"Large"
"Large 1600"
"Large 2048"
```

Is there a setting an author can choose to make "Original" label show up?

I'm aware there is a `-s` setting and it gets the job done, but it seems having a default `Original` no longer cuts it, so maybe I'd be worth adding an interactive choice or making size setting required.

**Comment by lmammino** [comment]
I agree with you that if `Original` is not available anymore then we should do a bit of research and see if there's a way to have a more stable default.

My first thought would be to create a list of "prioritised defaults" an take the first one available. I'll try to find the documentation to see if there's an official list of all the supported sizes.

**Comment by lmammino** [comment]
Unfortunately the Flickr API is not very clear on the possible size types. On thing I noticed though is that in every result it seems that the bigger sizes are placed at the end of the array when calling [`flickr.photos.getSizes`](https://www.flickr.com/services/api/flickr.photos.getSizes.html). So maybe we can use this heuristic to always pick the last type as default.
I will try to allocate some time to figure out how to implement this in the current code base but feel free to suggest a PR if you some time.
